querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -467,6 +467,7 @@ class DataDocQueryCellComponent extends React.PureComponent<IProps, IState> {
             element: ElementType.RUN_QUERY_BUTTON,
             aux: {
                 lintError: this.state.hasLintError,
+                sampleRate: this.sampleRate,
```

Comment:
could we only log sampleRate when sample rate is actually used? there could be a case where user selects sampling table and a rate, and then switch the table with no sample right?